docs/api/routing.md

```diff
@@ -63,6 +63,90 @@ app.get('/posts/:id/comment/:comment_id', async (c) => {
 })
 ```
 
+## Routing Priority
```

Comment:
The description is perfect. It's clear!

But I think you should not move this section to the above. The current position below of "Routing with host Header value" is good. This "Routing Priority" is important, but the user should read it after understanding basic routing syntaxes. It's my opinion.

![CleanShot 2025-04-15 at 10 51 18@2x](https://github.com/user-attachments/assets/d44d7f98-b1e1-4ca4-a5da-e5ec117ed8db)